CMSgov/mint-app
# [NOREF] Rename job codes and related auth/principal function/property names

# NOREF

## Changes and Description

- Rename hard-coded job codes to better reflect what we'll create with the IDM team
- Rename GQL roles to match job code names (this doesn't have to happen, but is somewhat nice IMO)



## How to test this change

1. Use `scripts/dev up` to start up MINT.
2. Login (local auth or the EZOD account, for that matter).
3. Navigate to http://localhost:8085/user-diagnostics to see information about the user.
4. Try different GQL queries/mutations that require different roles.

## What's left to do / open questions
- Should the frontend allow access to Secure routes if `isBasicUser` is `false`? It seems there were some assumptions around "successful Okta auth" being the same as "you're a basic user", but the presence of a job code would make me think that's not a good thing to assume - CC @patrickseguraoddball @StevenWadeOddball 

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
